### ISSUE TITLE
I do not see the individual data of all my test site individuals - 2.0.0

### ISSUE THREAD

**Issue by melizaa** [fisharebest/webtrees#1986]
I do not see the indi data of all my site individuals in the test systems. These individuals are for instance my maternal grandparents (I62, I63).          

The URL is   /index.php?route=individual&xref=I63&ged=Gedcom.ged     
I see the name, sex and main picture and the indi tabs.      
I see the individual tabs without any data.     
If I hit the Facts and events tab, I see some or all of the individual's facts and events.     
The other tabs do not show anything.          

I see in the Website log for <none> family tree error     
Exception: getCloseRelationshipName() called on unrelated individuals in   file /app/Functions/Functions.php on line 374 Stack trace: 1.   Exception->() /app/Functions/Functions.php:374 2. Fisharebest\Webtrees\Functions\Functions->getRelationship()   /app/Functions/Functions.php:222 3.   Fisharebest\Webtrees\Functions\Functions->getCloseRelationshipName()   /app/Functions/FunctionsPrintFacts.php:531 4.   Fisharebest\Webtrees\Functions\FunctionsPrintFacts->formatAssociateRelationship()   /app/Functions/FunctionsPrintFacts.php:358 5.   Fisharebest\Webtrees\Functions\FunctionsPrintFacts->printFact()   /resources/views/modules/personal_facts/tab.phtml:25 6. include()   /app/View.php:136 7. Fisharebest\Webtrees\View->render() /app/View.php:176   8. Fisharebest\Webtrees\View->make() /app/Helpers/functions.php:84 9.   view() /app/Module/IndividualFactsTabModule.php:130 10.   Fisharebest\Webtrees\Module\IndividualFactsTabModule->getTabContent() /resources/views/individual-page.phtml:109   11. include() /app/View.php:136 12. Fisharebest\Webtrees\View->render()   /app/View.php:176 13. Fisharebest\Webtrees\View->make()   /app/Helpers/functions.php:84 14. view()   /app/Http/Controllers/AbstractBaseController.php:199 15.   Fisharebest\Webtrees\Http\Controllers\AbstractBaseController->viewResponse()   /app/Http/Controllers/IndividualController.php:149 16.   Fisharebest\Webtrees\Http\Controllers\IndividualController->show()   [internal]:0 17. ReflectionMethod->invoke() /app/Resolver.php:87 18.   Fisharebest\Webtrees\Resolver->dispatch() /index.php:326 19. {closure}()   /app/Http/Middleware/CheckForMaintenanceMode.php:48 20.   Fisharebest\Webtrees\Http\Middleware\CheckForMaintenanceMode->handle()   /index.php:321 21. {closure}() /app/Http/Middleware/PageHitCounter.php:106   22. Fisharebest\Webtrees\Http\Middleware\PageHitCounter->handle()   /index.php:321 23. {closure}() /app/Http/Middleware/Housekeeping.php:70 24.   Fisharebest\Webtrees\Http\Middleware\Housekeeping->handle() /index.php:321   25. {closure}() /index.php:329

**Comment by melizaa** [comment]
My grandfather has a 2 _ASSO link to a young Austrian Refugee, who lived in my grandparents' house during WWII.  I see my grandfather's individual data when I remove this link.  I did not have a 3 RELA. 
I have tried to add the 3 RELA, but I get still the error.
I have many 2 ASSO links. I would not like to remove them.

If I remove the 1 FAMC record that links my grandmother to her parents' family F42, then I see her individual data?!?
